### DISCUSSION TITLE
How to configure TodoTelescope

### DISCUSSION THREAD

**Discussion by lcavanzo** [folke/todo-comments.nvim#232] [Q&A]
Hello, I've been trying to configure todo-comments with telescope, but I'm receiving an error while it tried to load the extension, how could I load the TodoTelescope extension?

would be great to show and example of configuration in the README :)

![CleanShot 2023-10-11 at 00 24 16](https://github.com/folke/todo-comments.nvim/assets/27904815/929d18af-17da-4d4e-9514-ad6e4cd49aca)

**Comment by tony-sol** [comment]
Show your configs, pls

Here's mine and it works well

```lua
-- ~/.config/nvim/lua/plugins/telescope.lua
return {
	'nvim-telescope/telescope.nvim',
...
	config       = function()
		local telescope = require('telescope')
		telescope.setup {
...
			extensions = {
...
				['todo-comments']  = {},
			},
		}
...
		telescope.load_extension('todo-comments')
	end
}

-- ~/.config/nvim/lua/keymaps.lua
keymap.set(
	{ 'n' },
	'<leader>tT',
	function()
		require('telescope').extensions['todo-comments'].todo()
	end,
	{ desc = 'Telescope: Todo-Comments' }
)

```
